Src/coffeescript/cql-execution/test/elm/comparison/test.coffee

```diff
@@ -75,6 +75,25 @@ describe 'Less', ->
   it 'should be true for 5 < 6', ->
     @aLtB_Int.exec(@ctx).should.be.true()
 
+  it 'should be true for 5 m < 4 m', ->
```

Comment:
And it looks like similar mistakes are repeated below.  The tests are actually right, but the string descriptions don't always match what the test is asserting.